charts/stable/dnsmadeeasy-webhook/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.2.0
 description: Cert-Manager Webhook for DNSMadeEasy
 name: dnsmadeeasy-webhook
-version: 3.2.0
+version: 3.2.1
```

Comment:
This might actually qualify as a 3.3.0 version, since the certmanager addition is a non-breaking additional functionality.

Reply:
changed to 3.3.0